src/widgets/helper/ChannelView.cpp

```diff
@@ -1098,8 +1098,10 @@ void ChannelView::drawMessages(QPainter &painter)
 
 void ChannelView::wheelEvent(QWheelEvent *event)
 {
-    if (event->orientation() != Qt::Vertical)
+    if (event->angleDelta().x())
```

Comment:
```suggestion
    if (!event->angleDelta().y())
```

If we only care about the vertical scroll value, maybe it makes more sense to check if there's any vertical scroll value rather than checking for the absence of a horizontal scroll value